fossology/uploads.py

```diff
@@ -36,10 +36,10 @@ def detail_upload(
         :Examples:
 
         >>> # Wait up to 20 minutes until the upload is ready
-        >>> long_upload = detail_upload(1, 120)
+        >>> long_upload = detail_upload(1, None, 120)
```

Comment:
Since `None` is the default for the group argument, using `detail_upload(1, wait_time=120)` should work too.

Reply:
Agreed.